packages/babel-plugin-transform-template-literals/test/fixtures/default/cache-revision/input.js

```diff
@@ -0,0 +1,11 @@
+var tag = v => v;
+
+function foo() {
+  return tag`some template`;
+}
+function bar() {
+  return tag`some template`;
+}
+assert.equal(foo(), foo());
+assert.equal(bar(), bar());
+assert.notEqual(foo(), bar());
```

Comment:
Could you please test the output? A bug could cause `assert.equal(0, 0);` for ex.

Reply:
Done.